notebooks/examples/python/Mosaic + Sedona Raster 1.2.x Example.py

```diff
@@ -0,0 +1,731 @@
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC > Get the normalized difference index between the extracted bands
+
+# COMMAND ----------
+
+df_nd = (
+    df_t1.selectExpr(
+        "origin", 
+        "geom",
+        "width", 
+        "height", 
+        "data", 
+        "bands", 
+        "RS_NormalizedDifference(band2, band1) as normalizedDifference"
+    )
+)
+display(df_nd)
+
+# COMMAND ----------
+
+# MAGIC %md > `RS_Append()` takes the data array containing bands, a new band to be appended, and number of total bands in the data array. It appends the new band to the end of the data array and returns the appended data-- __Not available in 1.2.0 from maven; IS available in 1.2.1 from master.__
+
+# COMMAND ----------
+
+df_append = (
+    df_nd.selectExpr(
+        "origin", "geom", 
+        "RS_Append(data, normalizedDifference, bands) as data_edited", 
+        "height", "width", "bands"
+    )
+    .drop("data")
+    .withColumn("nBand_edited", col("bands") + 1)
+    .drop("bands")
+)
+display(df_append)
+
+# COMMAND ----------
+
+# MAGIC %md _Could write GeoTiff DataFrame as GeoTiff image similar as before_
+# MAGIC 
+# MAGIC Example
+# MAGIC 
+# MAGIC ```
+# MAGIC (
+# MAGIC df_append.coalesce(1)
+# MAGIC   .write.mode("overwrite").format("geotiff")
+# MAGIC     .option("writeToCRS", "EPSG:4326")
+# MAGIC     .option("fieldGeometry", "geom")
+# MAGIC     .option("fieldNBands", "nBand_edited")
+# MAGIC     .option("fieldData", "data_edited")
+# MAGIC   .save(SAVE_DIR)
+# MAGIC )
+# MAGIC ```
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## User can also create some UDF manually to manipulate Geotiff dataframes
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC > Sample UDF calculates sum of all the values in a band which are greater than 1000.0
+
+# COMMAND ----------
+
+def SumOfValues(band):
+    total = 0.0
+    for num in band:
+        if num>1000.0:
+            total+=1
+    return total
+
+df_u1 = (
+    spark.read.format("geotiff").option("dropInvalid",True).option("readToCRS", "EPSG:4326").load(DATA_DIR)
+    .selectExpr("image.origin as origin","ST_GeomFromWkt(image.geometry) as Geom", 
+                "image.height as height", "image.width as width", "image.data as data", "image.nBands as bands")
+    .selectExpr("RS_GetBand(data,1,bands) as targetband", 
+                "height", "width", "bands", "Geom")
+)
+    
+calculateSum = udf(SumOfValues, DoubleType())
+spark.udf.register("RS_Sum", calculateSum)
+
+df_sum = df_u1.selectExpr("RS_Sum(targetband) as sum")
+display(df_sum)
+
+# COMMAND ----------
+
+# MAGIC %md > Sample UDF to visualize a particular region of a GeoTiff image
+
+# COMMAND ----------
+
+def generatemask(band, width,height):
+    for (i,val) in enumerate(band):
+        if (i%width>=12 and i%width<26) and (i%height>=12 and i%height<26):
+            band[i] = 255.0
+        else:
+            band[i] = 0.0
+    return band
+
+maskValues = udf(generatemask, F.ArrayType(DoubleType()))
```

Comment:
The ArrayType is not a member of functions. Please replace `F.ArrayType` with `ArrayType`